plugins/auth-backend/src/lib/catalog/CatalogIdentityClient.ts

```diff
@@ -61,9 +61,7 @@ export class CatalogIdentityClient {
     }
 
     // TODO(Rugvip): cache the token
```

Comment:
Can remove this TODO now, as it wouldn't be up to this implementation to do the caching